lib/renderer/renderer.ex

```diff
@@ -0,0 +1,115 @@
+defmodule Tabula.Renderer do
+
+  import Enum, only: [
+    concat: 2,
+    intersperse: 2,
+    join: 1,
+    map: 2,
+    max: 1,
+    with_index: 1,
+    zip: 2
+  ]
+
+  @index "#"
+
+  @sheets \
+  org_mode: [
+    heading:        " | ",
+    heading_border: "-+-",
+    row:            " | ",
+    spacer:         "-"
+  ],
+  github_md: [
+    heading:        " | ",
+    heading_border: " | ",
+    row:            " | ",
+    spacer:         "-"
+  ]
+
+  @default_sheet :org_mode
+
+  def run(rows, [_|_]=cols, opts) do
```

Comment:
`run` implies doing something with/for side effects. `render`, although redundant, would fit here better imho - it also conveys some meaningful action.